src/api_manager/auth/lib/auth_jwt_validator_test.cc

```diff
@@ -200,6 +200,37 @@ const char kPublicKeyX509[] =
     "L9Lb4gPPSUMSKy7Zy4Sn4il1U\\nFc94Mi9j13oeGvLOduNOStGu5XROIxDtCEjjn2y2SL2bPw"
     "0qAlIzBeniiApkmYw/\\no6OLrg==\\n-----END CERTIFICATE-----\\n\"}";
 
+
+// Please see jwt_and_jwk_generator.py under /src/tools/ for Public/private ES256
+// key pair, ES256-signed jwt token, and public jwk generation.
```

Comment:
Also add the command line to show how you generate public key and token using hte tool?